.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "openmaptiles"]
+	path = openmaptiles
+	url = https://github.com/tbicr/openmaptiles.git
```

Comment:
can you move this one to osm-by org?

Reply:
Sound good for me, but I don't have permissions to do that

Reply:
check now?

Reply:
moved